assets/tests/components/swingsView.test.tsx

```diff
@@ -1,7 +1,7 @@
-import { mount } from "enzyme"
```

Comment:
Note that I've also made updates to `swingsView.test.tsx` in #1725 (and as part of that converted the old Enzyme tests to RTL). I had to do some substantial other changes because the other changes in that PR changed some stuff about hooks and how the page renders, so there probably isn't a ton of overlap between our changes (though I haven't compared them line-by-line yet).

Reply:
Ah I forgot about that! I can remove it from this PR